backend/src/index.ts

```diff
@@ -112,3 +112,20 @@ app.post("/login", async (req: TypedRequest<LoginBody>, res: Response) => {
     },
   );
 });
+// curl -H 'Content-Type: application/json' -d '{ "email": "ben", "color": "pink"}' -X POST http://localhost:3000/subscribe
+
+app.get('/startGame', async (req: ParamsType<{roundCount: number}>, res) => {
+
+    const getDoc = await getDocs(collection(db, 'test_locations'));
+    const allLocations = getDoc.docs.map(doc => doc.data);
+
+    const n = req.params.roundCount;
+    // Shuffle array
+    const shuffled = allLocations.sort(() => 0.5 - Math.random());
+
+    // Get sub-array of first n elements after shuffled
+    let selected = shuffled.slice(0, n);
+
+    res.status(200).json(selected);
```

Comment:
ideally this endpoint would also create a game object in the database (see the interfaces file for what that would look like). lmk if u need any help w this